### DISCUSSION TITLE
Expose Niko doorbell to Homekit

### DISCUSSION THREAD

**Discussion by jonassmets** [joleys/niko-home-control-II#188] [Q&A]
Would it be possible to have the Niko video doorbell as a doorbell in Homekit? True HAS? This would make the Niko app unnecessary in my case?

**Comment by tijsverkoyen** [comment]
I don't know if it can be exposed to HomeKit. But check https://github.com/joleys/niko-home-control-II?tab=readme-ov-file#robinsip-videodoorstation

**Comment by jonassmets** [comment]
@tijsverkoyen I got the robinson video doorstation exposed in Home Assistant. By creating a static route from my Unifi Dream Machine to NHC2. So the IP of the video doorstation because visible in my regular network. I can see the camera, stills, and I think I can hear the audio.  But from what I understand, the Robinson doorstation uses a SIP protocol for 2 way audio? Do you know if there is an other way to get the 2 way audio working in Home Assistant? My final goal is to get a native doorbell in Homekit? With rich notifications, 2 way audio, and a way to unlock the door. 
love to hear from you.
Jonas

**Reply by tijsverkoyen** [reply]
To be honest I don't have a clue.

**Reply by patrickgoergen1** [reply]
https://www.reddit.com/r/homeautomation/comments/1vwydw6/comment/p5kkowy/?screen_view_count=2&ext-referrer=SEO
